packages/core/core-v2/src/Parcel.js

```diff
@@ -112,72 +108,74 @@ export default class Parcel {
     }
   }
 
-  async updateGraph() {
-    // Process all changes that may update the middle of the the graph
-    this.updateQueue.start();
-    await this.updateQueue.onIdle();
-    this.updateQueue.pause();
+  async updateGraph({signal}: BuildOpts) {
+    for (let [id, node] of this.graph.invalidNodes) {
```

Comment:
Where do nodes ever get removed from `invalidNodes`? I couldn't find it.

Reply:
Woops, good catch